transformations/concat_monolingual/transformation.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+from checklist.perturb import Perturb
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class ConcatMonolingual(SentenceOperation):
```

Comment:
I'm not sure about the goal of having a monolingual concat transformation. Is it intended for classification tasks?

I can understand that concating random source + translation might work as an augmentation method where model can learn from more combinations of data. However, for the monolingual concat, if only the source is concatenated with a random sentence, but the target (either a label in the classification task or a sentence in translation task) remains unchanged, how would it help the model? I would imagine there can be a meaning mismatch in this way.